test/fixtures/schema.sql

```diff
@@ -2185,3 +2185,61 @@ create table private.rollen (
   foreign key (film_id) references test.filme(id),
   foreign key (rolle_id) references test.schauspieler(id)
 );
+
+-- Tables used for testing embedding between partitioned tables
+
+DO $do$BEGIN
+    -- partitioned tables using the PARTITION syntax are supported from pg v10
+    IF (SELECT current_setting('server_version_num')::INT >= 100000) THEN
```

Comment:
Wow, this feature is really complete!

Let's keep all the identifiers as lowercase for consistency:

```suggestion
    -- partitioned tables using the PARTITION syntax are supported from pg v10
    if (select current_setting('server_version_num')::int >= 100000) then
```